python-package/xgboost/training.py

```diff
@@ -282,6 +284,11 @@ def cv(params, dtrain, num_boost_round=10, nfold=3, metrics=(),
     results = []
     cvfolds = mknfold(dtrain, nfold, params, seed, metrics, fpreproc)
     for i in range(num_boost_round):
+        if print_every_n:
+            if i == 0 or i % print_every_n == 0:
+                show_progress = True
+            else:
+                show_progress = False
```

Comment:
Personally feel  ``show_progress`` should be prioritized, for example what should happen ``show_progress=True`` and ``print_every_n=5``?